spec/restricted_input_spec.rb

```diff
@@ -15,7 +15,9 @@
 
     it 'only allows digits' do
       input = find '#credit-card-number'
-      input.send_keys 'abcdefghhijklmnopqrstuvwxyz !@#$%^&*()_=+ 1234567890123456'
+      input.send_keys 'abc'
```

Comment:
No, actually. In earlier iterations of this PR, some browsers would fail this test unless they were split up. I'm still not 100% sure why.